agent/app/agent_capability.go

```diff
@@ -55,6 +55,7 @@ const (
 	capabilityFirelensLoggingDriver             = "logging-driver.awsfirelens"
 	capabilityFirelensConfigFile                = "firelens.options.config.file"
 	capabilityFirelensConfigS3                  = "firelens.options.config.s3"
+	capabilityFullTaskSync                      = "full-sync"
```

Comment:
why not `full-task-sync` ?

Reply:
Backend named this before us, they decided on full-sync. 🙂